docs/fetchdependencies.sh

```diff
@@ -39,6 +40,11 @@ cp dependencies-docfx/docfx-google-api-dotnet-client.json dependencies/google-ap
 
 (cd dependencies/google-api-dotnet-client;
  rm NuGet.config;
+ dotnet restore Src/Support/GoogleApisClient.sln;
```

Comment:
(Nothing to do now, but this will break once GoogleApisClient supports UWP)

Reply:
Righto. Will work out what we need to do at that point.